--- texmaker-3.1/latexhighlighter.h.bak	2011-09-22 13:44:02.821102939 -0400
+++ texmaker-3.1/latexhighlighter.h	2011-09-22 13:44:16.269027184 -0400
@@ -20,7 +20,7 @@
 #include <QColor>
 #include <QTextBlockUserData>
 //#include "latexeditor.h"
-#include "hunspell/hunspell.hxx"
+#include <hunspell/hunspell.hxx>
 
 class QTextDocument;
 class LatexEditor;
--- texmaker-3.1/spellerdialog.h.bak	2011-09-22 13:44:33.962927470 -0400
+++ texmaker-3.1/spellerdialog.h	2011-09-22 13:44:51.697827482 -0400
@@ -13,7 +13,7 @@
 
 #include "ui_spellerdialog.h"
 #include "latexeditor.h"
-#include "hunspell/hunspell.hxx"
+#include <hunspell/hunspell.hxx>
 
 #include <QTextCursor>
 
--- texmaker-3.1/configdialog.cpp.bak	2011-09-22 13:43:08.380409352 -0400
+++ texmaker-3.1/configdialog.cpp	2011-09-22 13:43:24.867316602 -0400
@@ -122,7 +122,7 @@
 QDir spelldir(QCoreApplication::applicationDirPath());
 #else
 #ifdef DEBIAN_SPELLDIR
-QDir spelldir(PREFIX"/share/myspell/dicts");
+QDir spelldir(PREFIX"/share/myspell");
 #else
 QDir spelldir(PREFIX"/share/texmaker");
 #endif
--- texmaker-3.1/latexeditorview.h.bak	2011-09-22 13:53:43.399809984 -0400
+++ texmaker-3.1/latexeditorview.h	2011-09-22 13:54:31.496535228 -0400
@@ -22,7 +22,7 @@
 #include "gotolinewidget.h"
 #include "replacewidget.h"
 #include "minisplitter.h"
-#include "hunspell/hunspell.hxx"
+#include <hunspell/hunspell.hxx>
 
 class LatexEditorView : public QWidget  {
    Q_OBJECT
Index: texmaker-3.2.2/latexeditor.h
===================================================================
--- texmaker-3.2.2.orig/latexeditor.h
+++ texmaker-3.2.2/latexeditor.h
@@ -33,7 +33,7 @@
 
 #include "latexhighlighter.h"
 #include "textblockselection.h"
-#include "hunspell/hunspell.hxx"
+#include <hunspell/hunspell.hxx>
 
 typedef  int UserBookmarkList[3];
 
Index: texmaker-3.3/texmaker.h
===================================================================
--- texmaker-3.3.orig/texmaker.h
+++ texmaker-3.3/texmaker.h
@@ -48,7 +48,7 @@
 #include "usertagslistwidget.h"
 #include "xmltagslistwidget.h"
 #include "logeditor.h"
-#include "hunspell/hunspell.hxx"
+#include <hunspell/hunspell.hxx>
 #include "browser.h"
 #include "pdfviewerwidget.h"
 #include "pdfviewer.h"
Index: texmaker-3.3.4/texmaker.pro
===================================================================
--- texmaker-3.4.1.orig/texmaker.pro
+++ texmaker-3.4.1/texmaker.pro
@@ -4,7 +4,7 @@ TARGET	 = texmaker
 QT += network \
       xml \
       webkit
-CONFIG	+= qt warn_off release
+CONFIG	+= qt warn_off release link_pkgconfig
 TEXMAKERVERSION=3.4.1
 DEFINES += TEXMAKERVERSION=\\\"$${TEXMAKERVERSION}\\\"
 DEFINES += HAVE_SPLASH
@@ -67,25 +67,6 @@ HEADERS	+= texmaker.h \
 	synctex_parser_utils.h \
 	usertagslistwidget.h \
 	addtagdialog.h \
-	hunspell/affentry.hxx \
-	hunspell/affixmgr.hxx \
-	hunspell/atypes.hxx \
-	hunspell/baseaffix.hxx \
-	hunspell/csutil.hxx \
-	hunspell/dictmgr.hxx \
-	hunspell/hashmgr.hxx \
-	hunspell/htypes.hxx \
-	hunspell/hunspell.hxx \
-	hunspell/hunspell.h \
-	hunspell/langnum.hxx \
-	hunspell/license.hunspell \
-	hunspell/phonet.hxx \
-	hunspell/suggestmgr.hxx \
-	hunspell/license.myspell \
-	hunspell/filemgr.hxx \
-	hunspell/hunzip.hxx \
-	hunspell/replist.hxx \
-	hunspell/w_char.hxx \
 	singleapp/qtlocalpeer.h \
 	singleapp/qtlockedfile.h \
 	singleapp/qtsingleapplication.h \
@@ -171,18 +152,6 @@ SOURCES	+= main.cpp \
 	synctex_parser_utils.c \
 	usertagslistwidget.cpp \
 	addtagdialog.cpp \
-	hunspell/affentry.cxx \
-	hunspell/affixmgr.cxx \
-	hunspell/csutil.cxx \
-	hunspell/dictmgr.cxx \
-	hunspell/hashmgr.cxx \
-	hunspell/hunspell.cxx \
-	hunspell/phonet.cxx \
-	hunspell/suggestmgr.cxx \
-	hunspell/utf_info.cxx \
-	hunspell/filemgr.cxx \
-	hunspell/replist.cxx \
-	hunspell/hunzip.cxx \
 	singleapp/qtlocalpeer.cpp \
 	singleapp/qtlockedfile.cpp \
 	singleapp/qtsingleapplication.cpp \
@@ -279,6 +248,7 @@ isEmpty( ICONDIR ) {
 INCLUDEPATH  += /usr/include/poppler/qt4
 LIBS         += -L/usr/lib -lpoppler-qt4
 LIBS         += -L/usr/lib -lz
+LIBS         += -lhunspell-1.3
 DEFINES += PREFIX=\\\"$${PREFIX}\\\"
 target.path = $${PREFIX}/bin
 
